docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+- If you were experiencing issues with modules installation you may want to run `pulp-manage-db`
+  command. One of the migrations fixed this issue that was caused by incorrect puppet module's name
+  interpretation. You also may want to re-publish repos to update the symlinks to the correct module
+  with the corect name. The migration will stdout information about the repos that need re-publish.
```

Comment:
If you remove the first sentence consider adding this change:  s/The migration/When you run `pulp-manage-db`, the migration/